rebar.config

```diff
@@ -31,7 +31,7 @@
                  ]}.
 
 {profiles, [
-    {test, [{deps, [meck]}]},
+    {test, [{deps, [{meck, {git, "https://github.com/OpenRiak/meck.git", {branch, "openriak-3.2"}}}]}]},
```

Comment:
Cloning your own version of meck seems to me the wrong way to go

Reply:
This is consistent with our approach everywhere, which is to clone all repos we depend on - only we didn't do it previously for test.

If we just get `meck` latest, as was - we're subject to time-expensive surprises when all our deps can no longer update.  If we tag to a version, then we still need to check upstream for critical updates.

There's no perfect solution, so I'm happy that in this case we should resolve with the same solution used for non-test dependencies.